auslib/web/admin/views/base.py

```diff
@@ -7,12 +7,18 @@
 import logging
 
 
+log = logging.getLogger(__name__)
+
+
 def requirelogin(f):
     def decorated(*args, **kwargs):
         username = request.environ.get('REMOTE_USER', request.environ.get("HTTP_REMOTE_USER"))
         if not username:
-            logging.warning("Login Required")
-            return Response(status=401)
+            log.warning("Login Required")
+            return problem(401, 'Unauthenticated', 'Login Required')
+        elif not dbo.isKnownUser(username):
```

Comment:
I think this is one of those bugs where we may not remember why we added this a year from now - could you add a comment with a brief explanation, and a pointer back to the bug to help us remember? :)